queries.sql

```diff
@@ -94,4 +94,80 @@ SELECT full_name, COUNT(name) FROM owners O
     LEFT JOIN animals A
     ON O.id = A.owner_id   GROUP BY full_name 
     ORDER BY COUNT(name) 
-    DESC LIMIT 1;
\ No newline at end of file
+    DESC LIMIT 1;
+
+
+
+    
+SELECT A.name FROM vets VE
+    LEFT JOIN visits VI
+    ON VE.id = VI.vet_id
+    LEFT JOIN animals A
+    ON VI.animal_id = A.id
+    WHERE VE.name = 'William Tatcher'
+    ORDER BY VI.date DESC
+    LIMIT 1;
+
+SELECT COUNT(VI.animal_id) FROM vets VE
+    LEFT JOIN visits VI
+    ON VE.id = VI.vet_id
+    WHERE VE.name = 'Stephanie Mendez'
+    GROUP BY VE.name;
+
+SELECT V.name, SP.name FROM vets V
+    LEFT JOIN specializations S
+    ON S.vet_id = V.id
+    LEFT JOIN species SP
+    ON S.species_id = SP.id;
+
+SELECT A.name FROM vets VE
+    LEFT JOIN visits VI
+    ON VE.id = VI.vet_id
+    LEFT JOIN animals A
+    ON VI.animal_id = A.id
+    WHERE VE.name = 'Stephanie Mendez' AND VI.date >= '2020-04-01' AND VI.date <= '2020-08-30';
+
+SELECT A.name FROM animals A
+    LEFT JOIN visits V
+    ON A.id = V.animal_id
+    GROUP BY A.name
+    ORDER BY COUNT(date) DESC
+    LIMIT 1;
+
+SELECT A.name FROM vets VE
+    LEFT JOIN visits VI
+    ON VE.id = VI.vet_id
+    LEFT JOIN animals A
+    ON VI.animal_id = A.id
+    WHERE VE.name = 'Maisy Smith'
+    ORDER BY VI.date DESC
```

Comment:
The ORDER BY VI.date should be in Ascending not descending.

Reply:
Thanks for your feedback, just solved